ramos/pool.py

```diff
@@ -1,13 +1,14 @@
+import abc
+
 from .compat import ImproperlyConfigured, get_installed_pools, import_string
 from .exceptions import InvalidBackendError
 
 
-class BackendPool(object):
-    """
-    BackendPool is an interface to get instances of backend types
-    """
-
-    backend_type = None
+class AbstractPool(abc.ABC):
+    @classmethod
+    @abc.abstractmethod
+    def classes_iterator(cls):
+        ...
```

Comment:
Shouldn't it be a `pass` statement here? I've never seen the `Ellipsis` beeing used in this context.

Reply:
Oh yes, it's a habit. I have seen use in other contexts by the community besides object slicing. It is interesting ... But I will change it to NotImplementedError

E.g

- https://pydantic-docs.helpmanual.io/usage/models/#required-fields
- https://github.com/robinhood/faust/blob/master/faust/agents/agent.py#L428

Reply:
A `pass` statement is enough, since `@abc.abstractmethod` will not enable the class to be instantiated without implementing this method and we don't need to support Python 2.

Reply:
Truth! I would switch to a pass, but I don't even need to, because I think I will implement @jairhenrique suggestion. Thanks @renanivo!